### DISCUSSION TITLE
old announcements atleast for all 2.02 versions

### DISCUSSION THREAD

**Discussion by vsulin** [scandum/tintin#69] [Ideas]
Hi!

I'm running a bit older version and was planning to upgrade, but as I can no longer find older announcements. It's impossible to figure out what might break. Possible to get them older announcements from the old board or its all gone?

/Me

**Comment by scandum** [comment]
Most changes are listed here:
https://github.com/scandum/tintin/blob/master/mods/igr.mods

Changes likely to cause issues with upgrading are listed here:
https://github.com/scandum/tintin/blob/master/NEWS
